src/Service/UploaderHelper.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+
+namespace App\Service;
+
+use Symfony\Component\HttpFoundation\File\UploadedFile;
+
+class UploaderHelper
+{
+    private $uploadsPath;
+
+    public function __construct(string $uploadsPath)
+    {
+        $this->uploadsPath = $uploadsPath;
+    }
+
+    public function uploadArticleImage(UploadedFile $uploadedFile): string
+    {
+        $destination = $this->uploadsPath.'/Logos';
```

Comment:
Why add a "Logos" directory into the uploadsPath, which is already Logos ? Your route will be uploads/Logos/Logos/ ?

Reply:
Oh yes, i follow the tutorials and that brings there but it changes nothing; I should have rename it differently